integrationtest/src/test/java/org/mapstruct/itest/tests/FullFeatureCompilationExclusionCliEnhancer.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.itest.tests;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import org.junit.jupiter.api.condition.JRE;
+import org.mapstruct.itest.testutil.extension.ProcessorTest;
+
+/**
+ * Adds explicit exclusions of test mappers that are known or expected to not work with specific compilers.
+ *
+ * @author Andreas Gudian
+ */
+public final class FullFeatureCompilationExclusionCliEnhancer implements ProcessorTest.CommandLineEnhancer {
+    @Override
+    public Collection<String> getAdditionalCommandLineArguments(ProcessorTest.ProcessorType processorType,
+        JRE currentJreVersion) {
+        List<String> additionalExcludes = new ArrayList<>();
+
+        // SPI not working correctly here.. (not picked up)
```

Comment:
This comes from earlier, I didn't write a single new line in this class. Just pulled it outside of the deleted `FullFeatureTest`. I actually have a better idea now with Jupiter instead of doing what we are doing here.